packages/utils/src/budgets-converter.js

```diff
@@ -40,7 +40,10 @@ function convertBudgetsToAssertions(budgets) {
     }
 
     for (const {resourceType, budget: maxNumericValue} of budget.resourceSizes || []) {
-      assertions[`resource-summary:${resourceType}:size`] = ['error', {maxNumericValue}];
+      assertions[`resource-summary:${resourceType}:size`] = [
```

Comment:
mind making just this part the PR so we can merge and publish ASAP? :)

we can discuss and fix the display units separately

Reply:
Sure :)

I'll update it a bit later today.